packages/headless-components/services/src/react/ServiceList.tsx

```diff
@@ -0,0 +1,467 @@
+    (props: { className?: string }, ref: React.Ref<HTMLDivElement>) => {
+      const context = React.useContext(ServiceContext);
+      if (!context) {
+        throw new ServiceComponentError(
+          'Service components must be used within a Service.Root component',
+        );
+      }
+      const { service } = context;
+      const { className } = props;
+
+      return (
+        <div
+          data-testid={TestIds.serviceName}
+          ref={ref as React.Ref<HTMLDivElement>}
+          className={className}
+        >
+          {service.name}
+        </div>
+      );
+    },
+  ),
+
+  Description: React.forwardRef<HTMLDivElement, { className?: string }>(
+    (props: { className?: string }, ref: React.Ref<HTMLDivElement>) => {
+      const context = React.useContext(ServiceContext);
+      if (!context) {
+        throw new ServiceComponentError(
+          'Service components must be used within a Service.Root component',
+        );
+      }
+      const { service } = context;
+      const { className } = props;
+
+      if (!service.description) return null;
+
+      return (
+        <div
+          data-testid={TestIds.serviceDescription}
+          ref={ref as React.Ref<HTMLDivElement>}
+          className={className}
+        >
+          {service.description}
+        </div>
+      );
+    },
+  ),
+
+  Price: React.forwardRef<HTMLDivElement, { className?: string }>(
+    (props: { className?: string }, ref: React.Ref<HTMLDivElement>) => {
+      const context = React.useContext(ServiceContext);
+      if (!context) {
+        throw new ServiceComponentError(
+          'Service components must be used within a Service.Root component',
+        );
+      }
+      const { service } = context;
+      const { className } = props;
+
+      if (!service.payment?.fixed?.price) return null;
+
+      return (
+        <div
+          data-testid={TestIds.servicePrice}
+          ref={ref as React.Ref<HTMLDivElement>}
+          className={className}
+        >
+          {service.payment?.fixed?.price.value}{' '}
+          {service.payment?.fixed?.price.currency}
+        </div>
+      );
+    },
+  ),
+
+  Duration: React.forwardRef<HTMLDivElement, { className?: string }>(
+    (props: { className?: string }, ref: React.Ref<HTMLDivElement>) => {
+      const context = React.useContext(ServiceContext);
+      if (!context) {
+        throw new ServiceComponentError(
+          'Service components must be used within a Service.Root component',
+        );
+      }
+      const { service } = context;
+      const { className } = props;
+
+      if (!service?.schedule?.availabilityConstraints?.durations?.[0])
+        return null;
+
+      return (
+        <div
+          data-testid={TestIds.serviceDuration}
+          ref={ref as React.Ref<HTMLDivElement>}
+          className={className}
+        >
+          {`${service?.schedule?.availabilityConstraints?.durations?.[0].minutes!} minutes`}
+        </div>
```

Comment:
We should not have text here. I'm suggesting to render the duration value only (like in https://github.com/wix-incubator/headless-components/blob/services-list-component/packages/headless-components/stores/src/react/ProductList.tsx#L423) and adding the text on the shadcn level using a prop